middlewares/hacker.middleware.js

```diff
@@ -534,6 +568,18 @@ function parseAccept(req, res, next) {
     next();
 }
 
+/**
+ * Sets req.body.status to Accepted for next middleware.
+ * @param {{params:{email: string}, body: *}} req 
+ * @param {*} res 
+ * @param {*} next 
+ */
+function parseAcceptEmail(req, res, next) {
+    req.body.hacker.status = Constants.General.HACKER_STATUS_ACCEPTED;
```

Comment:
Is there a reason this is req.body.hacker.status, i think the update middleware uses req.body.status right?

Reply:
agreed, the comment says req.body.status ^^ Double check it unless you changed it.